packages/components/psammead-radio-schedule/src/ProgramCard/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,1106 @@
+}
+
+.c11 {
+  padding-left: 0.5rem;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c2 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c2 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c4 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c4 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c6 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c6 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c7 {
+    font-size: 0.9375rem;
+    line-height: 1.125rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c7 {
+    font-size: 0.875rem;
+    line-height: 1.125rem;
+  }
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c8 {
+    font-size: 0.75rem;
+    line-height: 1rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c8 {
+    font-size: 0.75rem;
+    line-height: 1rem;
+  }
+}
+
+<div
+  class="c0"
+>
+  <div
+    class="c1"
+  >
+    <h3
+      class="c2"
+    >
+      <a
+        class="c3"
+        href="/news/articles/cn7k01xp8kxo"
+      >
+        <span
+          class=""
+          role="text"
+        >
+          <span
+            aria-hidden="true"
+            class="c4"
+          >
+            LIVE 
+          </span>
+          <span
+            class="c5"
+            lang="en-GB"
+          >
+            Live, 
+          </span>
+          <span
+            class=""
+          >
+            Could a computer ever create better art than a human?
+          </span>
+          <span
+            class="c5"
+          >
+            , 
+            13:00
+            , 
+          </span>
+          <span
+            class="c6"
+          >
+            29/01/1990
+          </span>
+        </span>
+      </a>
+    </h3>
+    <p
+      class="c7"
+    >
+      The critic, author, poet and TV host was known for his witty commentary on international television.
+    </p>
+  </div>
+  <div
+    class="c8"
+  >
+    <span
+      class="c9"
+    >
+      <svg
+        aria-hidden="true"
+        class="c10"
+        focusable="false"
+        height="12px"
+        viewBox="0 0 13 12"
+        width="13px"
+      >
+        <path
+          d="M9.021 1.811l-.525.525c.938.938 1.5 2.25 1.5 3.675s-.563 2.738-1.5 3.675l.525.525c1.05-1.087 1.725-2.55 1.725-4.2s-.675-3.112-1.725-4.2z"
+        />
+        <path
+          d="M10.596.199l-.525.562c1.35 1.35 2.175 3.225 2.175 5.25s-.825 3.9-2.175 5.25l.525.525c1.5-1.462 2.4-3.525 2.4-5.775s-.9-4.312-2.4-5.812zM6.996 1.511l-2.25 2.25H.996v4.5h3.75l2.25 2.25z"
+        />
+      </svg>
+    </span>
+    <time
+      class="c11"
+      dir="ltr"
+    >
+      <span
+        class="c5"
+      >
+         Duration 30,00 
```

Comment:
Do you know if we have the translation for 'duration' yet?

Reply:
We don't have translations for `duration` at the moment